docs/tutorial.md

```diff
@@ -93,7 +86,7 @@ const int  width = 224;
 // Preprocessing input image
 cv::Mat image_mat = cv::imread(argv[1], CV_LOAD_IMAGE_COLOR);
 image_mat =
-  crop_and_resize(std::move(image_mat), cv::Size(width, height));
+  resize(std::move(image_mat), cv::Size(width, height));
 std::vector<float> image_data = reorder_to_chw_and_subtract_imagenet_average(image_mat, scale);
```

Comment:
How about separating mean subtraction and reorder? It will be more readable. For mean subtraction, we can use OpenCV's function.